packages/grafana-api-clients/src/clients/rtkq/correlations/v0alpha1/endpoints.gen.ts

```diff
@@ -407,6 +407,12 @@ export type ObjectMeta = {
     Populated by the system. Read-only. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names#uids */
   uid?: string;
 };
+
+/*
```

Comment:
I know editing this is bad, but I wanted to provide the code on what I would like it to resolve to.